src/components/Collection/Sidebar.css

```diff
@@ -22,7 +22,7 @@
   & input {
     background-color: #eff0f4;
     border-radius: var(--borderRadius);
-
+    height: 36px; /* issue #907 */
```

Comment:
@erquhart The height was `36.8px` before, with the icon `9.4px` from the top. Does the rounding matter here to you?